src/utils/image.ts

```diff
@@ -0,0 +1,8 @@
+import env from "decentraland-gatsby/dist/utils/env"
+
+export function getImageUrl(imageUrl?: string | null) {
+  if (env("NEW_ROLLOUT") && imageUrl && imageUrl.startsWith("/images")) {
```

Comment:
I would ask if it start with `http://` or `https://`
```suggestion
  if (env("NEW_ROLLOUT") && imageUrl && (imageUrl.startsWith("https://") || imageUrl.startsWith("http://"))) {
```